modules/persistent-storage-csi-secrets-store-driver-overview.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-secrets-store-driver-overview_{context}"]
+= Overview
+
+{product-title} secrets are stored with just base64 encoding, which can be easily decoded. Etcd provides encryption at rest for these secrets, but when secrets are retrieved they are decrypted and presented to the user. If RBAC is not set up on your cluster properly, anyone with API access can retrieve or modify a secret, and so can anyone with access to etcd. Additionally, anyone who is authorized to create a pod in a namespace can use that access to read any secret in that namespace. 
```

Comment:
* I think the part about base64 being "easily decoded" is probably not good to say
* I'd go with "Kubernetes secrets" because I will be talking about the feature to sync Kubernetes secrets
* etcd is always lowercase

```suggestion
Kubernetes secrets are stored with Base64 encoding. etcd provides encryption at rest for these secrets, but when secrets are retrieved, they are decrypted and presented to the user. If role-based access control is not configured properly on your cluster, anyone with API or etcd access can retrieve or modify a secret. Additionally, anyone who is authorized to create a pod in a namespace can use that access to read any secret in that namespace.
```